scripts/shell.nix

```diff
@@ -6,6 +6,8 @@ in with pkgs;
       cabal-install
       hlint
       haskellPackages.apply-refact
+      haskellPackages.fourmolu
```

Comment:
is it possible to specify the version? We're using v0.12 in the CI, it wouldn't be nice for people to land on another version :(

Reply:
Would you mind if I changed the `shell.nix` to a flake. Its really hard to control the dependency versions using <nixpkgs> (it just uses the nixpkgs that the users system has in scope, instead of a project specific pin).